bootstrap/pesticide.py

```
@@ -1117,4 +1229,8 @@ def main() -> int:
 
 
 if __name__ == "__main__":
-    raise SystemExit(main())
+    try:
+        raise SystemExit(main())
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)